### DISCUSSION TITLE
Create roll only if single field not null and not both

### DISCUSSION THREAD

**Discussion by alwerr** [pocketbase/pocketbase#5972] [Q&A]
So i have collection with video and image upload.
I want to be able to allow only video or image when creating but nothing so far, but nothing so far.
Is there a way for it to work?
```
((@collection.posts.video=null && img != null) ||
(@collection.posts.video!=null && img = null)) 
```

**Comment by ganigeorgiev** [comment]
The above rule doesn't really make sense.

Please paste your collections configurations as json (Dashboard > Settings > Export collections), explain what collection you are targetting and share the used PocketBase version (_you can find it in the bottom-right corner in the Dashboard_).
